src/styles/theme.scss

```diff
@@ -61,6 +61,8 @@
 	--background-primary: #ffffff;
 	--background-secondary: #fbfbfb;
 
+	--background-primary-darker: hsl(0, 0%, 90%);
```

Comment:
I think it will be better to change colors like this to hex, because for the most of the code we use hex for color